climpred/create_climpred_data.py

```diff
@@ -0,0 +1,141 @@
+import glob as _glob
+import os as _os
+
+import numpy as np
+import xarray as xr
+
+from .constants import CLIMPRED_ENSEMBLE_DIMS
+
+
+def get_path(
+    dir_base_experiment='/work/bm1124/m300086/CMIP6/experiments',
+    member=1,
+    init=1960,
+    model='hamocc',
+    output_stream='monitoring_ym',
+    timestr='*1231',
+    ending='nc',
+):
+    """Get the path of a file of for MPI-ESM standard output file names and directory.
+
+    Args:
+        dir_base_experiment (str): Path of experiments folder. Defaults to
+        "/work/bm1124/m300086/CMIP6/experiments".
```

Comment:
FYI, if you go onto a new line you just need to add four spaces to the beginning to make sure the API auto-generator picks it up correctly. (You do not need to space in line with "Path" for instance)